src/components/marker/marker.helper.js

```diff
@@ -114,14 +114,10 @@ function getMarkerSize(config) {
   let medium = small + (MARKER_MEDIUM_OFFSET * config.maxZoom) / 3;
   let large = small + (MARKER_LARGE_OFFSET * config.maxZoom) / 3;
 
-  if (config.node && !config.node.viewGenerator) {
-    switch (config.node.symbolType) {
-      case CONST.SYMBOLS.CIRCLE:
-        small = 0;
-        medium = 0;
-        large = 0;
-        break;
-    }
+  if (config.node?.viewGenerator || SYMBOLS_WITH_OPTIMIZED_POSITIONING.includes(config.node?.symbolType)) {
```

Comment:
should be `has` instead of `includes` since `SYMBOLS_WITH_OPTIMIZED_POSITIONING` is a Set. Sandbox run fails otherwise.